samples/samples/snippets.py

```diff
@@ -349,23 +351,23 @@ def insert_data(instance_id, database_id):
             table="Singers",
             columns=("SingerId", "FirstName", "LastName"),
             values=[
-                (1, u"Marc", u"Richards"),
-                (2, u"Catalina", u"Smith"),
-                (3, u"Alice", u"Trentor"),
-                (4, u"Lea", u"Martin"),
-                (5, u"David", u"Lomond"),
+                (1, "Marc", "Richards"),
+                (2, "Catalina", "Smith"),
+                (3, "Alice", "Trentor"),
+                (4, "Lea", "Martin"),
+                (5, "David", "Lomond"),
             ],
         )
 
         batch.insert(
             table="Albums",
             columns=("SingerId", "AlbumId", "AlbumTitle"),
             values=[
-                (1, 1, u"Total Junk"),
-                (1, 2, u"Go, Go, Go"),
-                (2, 1, u"Green"),
-                (2, 2, u"Forever Hold Your Peace"),
-                (2, 3, u"Terrified"),
+                (1, 1, "Total Junk"),
+                (1, 2, "Go, Go, Go"),
+                (2, 1, "Green"),
+                (2, 2, "Forever Hold Your Peace"),
+                (2, 3, "Terrified"),
```

Comment:
Why were these changed from unicode strings to normal strings? Does having unicode strings affect the newly written samples in any way?

Reply:
This was automatically done by the linting application as per a new version